app/views/courses/age_range/_form_fields.html.erb

```diff
@@ -12,8 +12,8 @@
   <div class="govuk-radios__divider">or</div>
   <div class="govuk-radios__item">
     <%= form.radio_button :age_range_in_years, 'other', class: 'govuk-radios__input', data: {"aria-controls" => "other-container" },
-      checked: course.other_age_range? && course.age_range_in_years.present?,
-      aria: { describedby: @errors && @errors[:age_range_in_years] ? "course_age_range_in_years-error" : '' } %>
+      checked: (course.other_age_range? && course.age_range_in_years.present?) || (@errors && (@errors[:age_range_in_years_to].present? || @errors[:age_range_in_years_from].present?)),
```

Comment:
This is very ugly now.
@dankmitchell says this could be refactored once Rails 6.1 ships with components. In the meantime let me know if you have any short term suggestions or think it requires a comment

Reply:
probably wise just to slap a comment as a reminder

Reply:
How about just putting this behind an appropriately-named helper function?

Reply:
Like the helper function idea as a temporary measure 👍